src/app/forms/templates/trl/trl-purchaser.template.ts

```diff
@@ -10,64 +11,74 @@ export const TrlPurchasers: FormNode = {
       type: FormNodeTypes.GROUP,
       children: [
         {
-        name: 'name',
-        label: 'Name',
-        value: '',
-        type: FormNodeTypes.CONTROL
-      },
+          name: 'name',
+          label: 'Name',
+          value: '',
+          type: FormNodeTypes.CONTROL,
+          validators: [{ name: ValidatorNames.MaxLength, args: 150 }, { name: ValidatorNames.Required }]
+        },
         {
           name: 'address1',
           label: 'Building and street (box 1 of 2)',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          validators: [{ name: ValidatorNames.MaxLength, args: 60 }, { name: ValidatorNames.Required }]
         },
         {
           name: 'address2',
           label: 'Building and street (box 2 of 2)',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          validators: [{ name: ValidatorNames.MaxLength, args: 60 }, { name: ValidatorNames.Required }]
         },
         {
           name: 'postTown',
           label: 'Town or city',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          validators: [{ name: ValidatorNames.MaxLength, args: 60 }, { name: ValidatorNames.Required }]
         },
         {
           name: 'address3',
           label: 'County',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          validators: [{ name: ValidatorNames.MaxLength, args: 60 }]
         },
         {
           name: 'postCode',
           label: 'Postcode',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
```

Comment:
Could we add the variable width input boxes here to reflect the service design, if you check out the manufacturer template it's very similar, you just have to add a `width` attribute with the corresponding size to each node